jcl/src/jdk.incubator.foreign/share/classes/openj9/internal/foreign/abi/TypeLayoutCheckHelper.java

```diff
@@ -184,38 +184,38 @@ private static void validateValueLayoutKind(MemoryLayout targetLayout, Class<?>
 				.orElseThrow(() -> new IllegalArgumentException("The layout's ABI class is empty")); //$NON-NLS-1$
 		switch (kind) {
 		case CHAR:
-			/* the CHAR layout (8bits) in Java maps to bool or byte in C */
+			/* The CHAR layout (8bits) in Java maps to bool or byte in C. */
```

Comment:
Please add a space between `8` and `bits`, likewise for 16bits, 32bits, and 64bits.

Reply:
Fixed.